composer.json

```diff
@@ -1,39 +1,42 @@
 {
-  "name": "dunglas/php-property-info",
-  "type": "library",
-  "description": "Retrieve type and description of PHP properties using various sources ",
-  "keywords": [
-    "property",
-    "type",
-    "PHPDoc",
-    "symfony",
-    "validator",
-    "doctrine"
-  ],
-  "homepage": "http://dunglas.fr",
-  "license": "MIT",
-  "authors": [
-    {
-      "name": "Kévin Dunglas",
-      "email": "dunglas@gmail.com"
+    "name": "dunglas/php-property-info",
+    "type": "library",
+    "description": "Retrieve type and description of PHP properties using various sources ",
+    "keywords": [
+        "property",
+        "type",
+        "PHPDoc",
+        "symfony",
+        "validator",
+        "doctrine"
+    ],
+    "homepage": "http://dunglas.fr",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Kévin Dunglas",
+            "email": "dunglas@gmail.com"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "PropertyInfo\\": "src/PropertyInfo"
+        }
+    },
+    "require": {
+        "php": ">=5.4"
+    },
+    "require-dev": {
+        "phpspec/phpspec": "~2.1",
+        "doctrine/orm": "~2.3",
+        "phpdocumentor/reflection": "~1.0"
```

Comment:
So you would put `"phpdocumentor/reflection": "^1.0.3"` and keep conflict tag as well?

This is a little bit more complicated to maintain, isn't it? For sample, in case we have to bump conflict to `<1.0.6`, we have to update those two lines. Only one modification is needed with my case.

Thanks for your time. :+1: